changelog/unreleased/bugfixes/7567.md

```diff
@@ -0,0 +1 @@
+- Fix an issue where not all fields of incident were converted to JSON properly.
```

Comment:
Fixed an issue where some of the incident fields might have been lost after refresh.